src/main/java/org/eolang/jeo/representation/XmirRepresentation.java

```diff
@@ -64,11 +64,11 @@ private XmirRepresentation(final XmlDoc xml, final String source) {
     public String name() {
         final XmlNode root = this.xml.root();
         return new ClassName(
-            root.xpath("/program/metas/meta[head[text()]='package']/tail/text()")
+            root.xpath("/object/metas/meta[head[text()]='package']/tail/text()")
                 .stream()
                 .findFirst()
                 .orElse(""),
-            root.xpath("/program/@name").get(0)
+            root.xpath("/object/o/@name").get(0)
```

Comment:
@volodya-lombrozo `/object/o/@name` is not the same as `/program/@name`. The second one contained FQN with package. Please use `ObjectName` class from `eo-parser` as replacement for `/program/@name`

Reply:
@maxonfjvipon Thank you for the clue. Should we keep the both `/object/o/@name` and `/object/@name` names? The linter complained about this second one, actually. So, I removed it and left only the `/object/o/@name`.   

I've just checked the `ObjectName` class implementation and it's a bit suboptimal for this particular case. Moreover, the class lacks of tests. Thus, I don't trust it.

Reply:
@volodya-lombrozo `/object/@name` does not exist anymore

Reply:
@volodya-lombrozo after [this](https://github.com/objectionary/eo/pull/4114) is merged - there will be tests for `ObjectName`